purchases/src/main/kotlin/com/revenuecat/purchases/EntitlementInfo.kt

```diff
@@ -264,18 +264,18 @@ enum class OwnershipType {
     /**
      * The purchase was made directly by this user.
      */
-    @SerialName("purchased")
+    @SerialName("PURCHASED")
```

Comment:
Hmm I wonder, would it work without actually adding the `@SerialName` annotations?